src/main/webapp/i18n/de/error.json

```diff
@@ -88,6 +88,7 @@
         "solutionEntryError": "Fehler beim Laden der extrahierten Aufgaben aus der Aufgabenstellung.",
         "systemNotificationNeedsBothDates": "Sowohl das Mitteilungsdatum als auch das Ablaufdatum müssen gesetzt sein.",
         "systemNotificationNeedsNotificationBeforeExpiration": "Das Ablaufdatum muss nach dem Mitteilungsdatum liegen.",
-        "ltiNotSupported": "LTI wird auf dieser Artemis-Instanz nicht unterstützt"
+        "ltiNotSupported": "LTI wird auf dieser Artemis-Instanz nicht unterstützt",
+        "loadCourseUnnecessarily": "The course was already retrieved before and stored in the CourseStorageService. Are you sure retrieving the course again is necessary?"
```

Comment:
Why was this translation added? It seems unused. I also would not expect to see such a developer-aimed message somewhere in the webclient.

Reply:
Ah, missed that while I removed an unwanted feature in this PR